client/components/pages/login/PasswordEditionForm.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts" setup>
+import { object, string, ref as yref } from 'yup'
+import { useForm } from 'vee-validate'
+import { UPDATE_PASSWORD_FORM } from '~/utils/forms'
+import { ForcePasswordChangeDoneEvent } from '~/utils/events'
+
+const { t } = useI18n()
+const { logout } = useAuth()
+const updatePassword = useUpdatePassword()
+const { setUser } = useAuth()
+
+const loading = ref(false)
+const passwordField = ref<Nullable<CompElement>>(null)
+
+onMounted(() => passwordField.value?.$el.focus({ preventScroll: true }))
+
+const schema = object({
+  password: string()
+    .required(t('forms.change_password.errors.missing_password'))
+    .min(UPDATE_PASSWORD_FORM.password.min, t('forms.change_password.errors.password_too_short', { min: UPDATE_PASSWORD_FORM.password.min }))
+    .max(UPDATE_PASSWORD_FORM.password.max, t('forms.change_password.errors.password_too_long', { max: UPDATE_PASSWORD_FORM.password.max })),
+  confirmPassword: string()
+    .required(t('forms.change_password.errors.missing_password_confirmation'))
+    .oneOf([yref('password')], t('forms.change_password.errors.passwords_mismatch')),
+})
+const { handleSubmit } = useForm({
+  validationSchema: schema,
+})
+
+const onSubmit = handleSubmit(async (values) => {
```

Comment:
```suggestion
const onSubmit = handleSubmit((values) => {
```

(since no await in body)